### PR TITLE
feat: RFD 8 Public Roadmap

### PR DESCRIPTION
**Pull request by fargozhu**
## Description

The goal is to give the community a place where they can find the project features, their state and a potential timeline.

## Motivation and Context

Since KubeCF is the core of some of the downstream products, I do believe it's important to give a glimpse of what the team collective decided what is important to have and/or explore, based on community feedback and suggestions.

Note: it's not a place for public discussion (since that should have happen before we reach this point)!

### CODE REVIEW

File: doc/rfd/rfd/0008/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+* Beta: public available but for testing and gather feedback.
+* GA: public available.
+
+Some features may be allocated to a desired timeframe but still in exploratory phases, which can
+lead to potential delays, since their main purpose is to gather community feedback.
+
+* in design: we have decided to build but still figuring out how.
+* exploring: we are considering building, but we need more feedback to make a final decision.
+
+Any feature can be abandoned by team decision, independently of it state and the reason well
```
Comment (viccuad) [inline doc/rfd/rfd/0008/README.md:39]:
```suggestion
Any feature can be abandoned by team decision independently of its state, provided there's a reason well
```

**Review by viccuad — DISMISSED** [review]

File: doc/rfd/rfd/0008/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+particular feature.
+
+We also have two labels: *in design* or *exploring* which in conjunction with the timeframe column
+can indicate how far are we with that particular feature and it may give a better sense if it can be
+released within the initial timeframe.
+
+## Disclaimer
+
+This roadmap does not represent any commitment, promise or obligation to deliver any feature or any
+feature on a particular date. The main goal is to give the community the team high level development
```
Comment (andreas-kupries) [inline doc/rfd/rfd/0008/README.md:54]:
```suggestion
feature on a particular date. The main goal is to show the community the team's high level development
```

File: doc/rfd/rfd/0008/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+plan, so please do not rely on this roadmap to make any internal decision.
```
Comment (andreas-kupries) [inline doc/rfd/rfd/0008/README.md:55]:
```suggestion
plan. Please __do not__ rely on this roadmap to make any internal decision.
```

File: doc/rfd/rfd/0008/README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+* in design: we have decided to build but still figuring out how.
+* exploring: we are considering building, but we need more feedback to make a final decision.
+
+Any feature can be abandoned by team decision independently of its state, followed by a
+well-documented reason.
+
+## Stages
+
+The roadmap is arranged in quarters columns to give an idea when is our intention to release that
+particular feature.
```
Comment (andreas-kupries) [inline doc/rfd/rfd/0008/README.md:45]: Rephrased:
```
The roadmap is arranged in columns, each representing a quarter year.
The contents indicate our intent on when to release any particular feature.
Note that an intent is not a promise.
```

**Review by andreas-kupries — CHANGES_REQUESTED** [review]
Lots of general re-phrasing for clarity.

File: doc/rfd/rfd/0008/README.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+## Phases
+
+Every project feature has it's own lifecycle.
+
+* Beta: public available but for testing and gather feedback.
+* GA: public available.
+
+Some features may be allocated to a desired timeframe but still in exploratory phases, which can
+lead to potential delays, since their main purpose is to gather community feedback.
```
[the remark this answers is not in the document]
Comment (fargozhu) [reply doc/rfd/rfd/0008/README.md:34]: Even for me ;)

I am rephrasing it....

File: doc/rfd/rfd/0008/README.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+state: discussion
+discussion: https://github.com/cloudfoundry-incubator/kubecf/issues?q=%22RFD+8%22+
+---
+
+# RFD 0008 Public Roadmap
+
+## Motivation
+
+The *Roadmap* project can be used used to track the features that the team decided to engage with,
+which phase they are and when they may come available.
```
Comment (andreas-kupries) [inline doc/rfd/rfd/0008/README.md:12]:
```suggestion
which phase they are in and when they may come available.
```

File: doc/rfd/rfd/0008/README.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+* in design: we have decided to build but still figuring out how.
+* exploring: we are considering building, but we need more feedback to make a final decision.
+
+Any feature can be abandoned by team decision independently of its state, followed by a
+well-documented reason.
+
+## Stages
+
+The roadmap is arranged in columns, each representing a quarter year, and it indicates when are we
+intent to release a feature.
```
Comment (andreas-kupries) [inline doc/rfd/rfd/0008/README.md:45]:
```
- ... it indicates when are we intent to release a feature.
+ .. it indicates when we intend to release a feature.
```
`intent` is the thing/noun, `(to) intend` the action/verb.

**Review by andreas-kupries — CHANGES_REQUESTED** [review]
Nits left.

**Comment by viovanov** [comment]
👍

**Pull request merged**